src/python/pants/backend/experimental/terraform/register.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
We want to also have `backend/experimental/terraform/lint/tffmt/register.py`, and one for `validate`. Ack that that's lots of boilerplate - but we want it so that linters/formatters are _always_ explicitly opted into imo.  We had a lot of issues in V1 land that we automatically opted you in to a lot of functionality that was hard to disable.

The `backend/terraform` folders should use that structure too, please.

Reply:
Done.